src/models.ts

```diff
@@ -17,6 +17,9 @@ interface ReleaseAttributesBase {
 	source: string;
 	start_timestamp: Date;
 	end_timestamp?: Date;
+	semver?: string;
```

Comment:
I think that's not nullable, right?

Reply:
But we said the user could not provide a value and the API would set the default to "0.0.0", no? If we make it non-nullable this will break the CLI

Reply:
Oh you are right, I just saw how this is used, sorry.
In that sense the is_final should also be optional.
Based on how these are used, maybe we can remove semver & is_final from here and add them as optional in ReleaseAttributes and as non-optional in ReleaseModel.
What do you think?
PS: Ideally I would prefer to have these fields as mandatory here and have a type helper to mark them as optional in ReleaseAttributes, but that sounds like an overkill for this PR.

Reply:
Oh, I see what you mean. Yes, that makes total sense as it makes the types more accurate. Making the change now

Reply:
done!